src/components/slider/slider.tsx

```diff
@@ -0,0 +1,41 @@
+import { default as RNSlider, SliderProps } from '@react-native-community/slider';
+import React, { FC } from 'react';
+import { View } from 'react-native';
+
+import { greyLight200, orangeLight200 } from '../../config/styles';
+import { SliderStyles } from './slider.styles';
+
+type Props = SliderProps & {
+  leftElementRender?: Element;
+  middleElementRender?: Element;
+  rightElementRender?: Element;
```

Comment:
There is no case when these elements will be different right now.
hardcode `Low Mid High` inside the `Slider`